### PR TITLE
DEVDOCS-4489: [update] Catalog, indicate that product name + SKU are unique

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-4489]

## What changed?
Updated the descriptions for name and SKU.

## Anything else?
Related PRs, salient notes, etc

[DEVDOCS-4489]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4489?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by vil11** [comment]
🌻 very nice, thnx

**Comment by markcmurphy** [comment]
Can you change the base branch to staging?

**Comment by bc-traciporter** [comment]
@markcmurphy Sorry about that, I changed it to Staging.

File: reference/catalog.v3.yml [hunk 1]
```diff
@@ -23711,7 +23711,7 @@ components:
           minLength: 0
           type: string
           description: |
-            User defined product code/stock keeping unit (SKU).
+            A unique user defined product code/stock keeping unit (SKU).
```
Comment (bc-andreadao) [inline reference/catalog.v3.yml:23714]:
```suggestion
            A unique user-defined product code (stock keeping unit).
```

**Review by bc-andreadao — COMMENTED** [review]
suggested a copyedit

**Review by stanzikratelbc — APPROVED** [review]
LGTM

File: reference/catalog.v3.yml (hunk 1 above)
Comment (stanzikratelbc) [inline reference/catalog.v3.yml:23714]: add a hyphen to say user-defined

**Review by stanzikratelbc — COMMENTED** [review]
Left one comment to address

**Review by slsriehl — APPROVED** [review]
LGTM, thanks everyone!

**Review by stanzikratelbc — COMMENTED** [review]
LGTM Ready to Merge

**Pull request merged**